llvm/docs/CommandGuide/llvm-symbolizer.rst

```diff
@@ -216,6 +224,12 @@ OPTIONS
   This can be used to perform lookups as if the object were relocated by the
   offset.
 
+.. option:: --skip-line-zero
+
+  If an address does not have an associated line number, use the last line
+  number from the current sequence in the line-table. Such lines are labeled
+  as "approximate" in the output as they may be misleading.
+
```

Comment:
Perhaps this documentation should be the same as the --help documentation in Opts.td? (maybe a few fewer words here, a few more there, but some convergence seems good)

Reply:
Matched both the documentation in Opts.td to llvm-symbolizer.rst.